person/models.py

```diff
@@ -24,10 +43,23 @@ class Person(AbstractUser):
     is_worker = models.BooleanField(default=False, verbose_name="¿Es trabajador?") #Extiende a clase WorkerProfile si es un trabajador
 
     class Meta:
-        db_table = 'person'
+        abstract = True
 
-class WorkerProfile(models.Model):
-    Person = models.OneToOneField(Person, on_delete=models.CASCADE)
-    #ONG = models.OneToOneField(ONG, on_delete=models.CASCADE) #ONG en la que trabaja
-    #active = models.BooleanField(default=True) #¿Sigue trabajando en la ONG?
+class GodFather(Person):
+    dni = models.CharField(max_length=9, unique=True,verbose_name='DNI')
+    payment_method = models.CharField(max_length=50, choices=PAYMENT_METHOD,verbose_name='Método de pago',)
+    bank_account_number = models.CharField(max_length=20,verbose_name='Número de cuenta bancaria', validators=[RegexValidator(r'^[0-9]+$')])
+    bank_account_holder = models.CharField(max_length=100,verbose_name='Titular de cuenta bancaria')
+    bank_account_reference = models.CharField(max_length=100,verbose_name='Referencia de cuenta bancaria', validators=[RegexValidator(r'^[0-9]+$')])
+    amount = models.DecimalField(max_digits=10, decimal_places=2,verbose_name='Cantidad',validators=[MinValueValidator(1)])
+    frequency = models.CharField(max_length=20, choices=FREQUENCY,verbose_name='Frecuencia de pago')
+    seniority = models.CharField(max_length=50,verbose_name='Antiguedad')
```

Comment:
_verbose_name_ of seniority should be "Antigüedad", note the umlaut.

Reply:
you are correct, i will change that